contrib/backporting/common.sh

```diff
@@ -32,15 +32,20 @@ get_remote () {
   echo "$remote"
 }
 
+get_user() {
+  gh_username=$(hub api user --flat | awk '/.login/ {print $2}')
```

Comment:
Nit: Could be nice to cache this in a global variable, to avoid curling again on subsequent calls. But it's certainly not blocking the current PR.